python/plugins/processing/algs/gdal/rasterize_over.py

```diff
@@ -90,7 +83,7 @@ def name(self):
         return 'rasterize_over'
 
     def displayName(self):
-        return self.tr('Rasterize (write over existing raster)')
+        return self.tr('Rasterize (write attribute value over existing raster)')
```

Comment:
A lengthy string like this should be moved to the shortDescription() method, and the name should be kept as brief as possible. Maybe "Rasterize (overwrite with attribute)"?